vm/runtime.ts

```diff
@@ -2097,8 +2224,21 @@ var AOTMD = J2ME.aotMetaData;
  * Are we currently unwinding the stack because of a Yield? This technically
  * belonges to a context but we store it in the global object because it is
  * read very often.
+ *
+ * We define it as a getter/setter to enable breaking when it is set in Chrome:
+ *   http://stackoverflow.com/questions/11618278/how-to-break-on-property-change-in-chrome
  */
-var U: J2ME.VMState = J2ME.VMState.Running;
+var _U: J2ME.VMState = J2ME.VMState.Running;
```

Comment:
This checked extremely often, can we remove this or enable it only for debugging?